project2/styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: rgb(255, 255, 255);
+    color: rgb(0, 0, 0);
+    font-family: 'Be Vietnam', sans-serif;
+}
+
+section {
+    display: inline-block;
+    margin: auto;
+    padding-bottom: 15px;
+    width: 400px;
+
+}
+
+h2,
+ul,
+p {
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+a:link,
+a {
+    color: black;
+    text-decoration: none;
+}
+
+a+a {
```

Comment:
This is specific to the `.readmore` links, so I would make sure these styles are applied only within that class.